bin/start-coordinator.sh

```diff
@@ -107,7 +107,8 @@ else
   GC_ARGS=$GC_LOG_ARGS_NEW
 fi
 
-$RUNNER ${UNIFFLE_COORDINATOR_JAVA_OPTS:-""} $ARGS $JVM_ARGS $GC_ARGS -cp $CLASSPATH $MAIN_CLASS --conf "$COORDINATOR_CONF_FILE" $@ &
+UNIFFLE_COORDINATOR_JAVA_OPTS=${UNIFFLE_COORDINATOR_JAVA_OPTS:-""}
```

Comment:
It seems you have to merge or rebase from master branch

Reply:
You are right. I used cherry-pick. Because I feel more convenient.